documents/decimal.go

```diff
@@ -17,6 +18,9 @@ const (
 	maxDecimalByteLength  = 32
 )
 
+// minDec is the min absolute value we support
+var minDec = decimal.RequireFromString("0.1")
```

Comment:
this can't be a constant?

Reply:
questions that arise for me are:
- minimum amount of an invoice or any field in the document is 0.01. Is that good enough?
- forces us to represent % in base100 + decimals, which i think is ok.

Reply:
Maybe I missed something. Let me think more on this